src/main/java/me/hydos/lint/world/feature/BetterTreeFeature.java

```diff
@@ -171,7 +141,7 @@ private int method_29963(TestableWorld testableWorld, int i, BlockPos blockPos,
 			for (int l = -k; l <= k; ++l) {
 				for (int m = -k; m <= k; ++m) {
 					mutable.set(blockPos, l, j, m);
-					if (!canTreeReplace(testableWorld, mutable) || !treeFeatureConfig.ignoreVines && isVine(testableWorld, mutable)) {
+					if (!TreeFeature.canTreeReplace(testableWorld, mutable) || !treeFeatureConfig.ignoreVines && TreeFeature.isVine(testableWorld, mutable)) {
```

Comment:
`TreeFeature.isVine` is private [according to the build](https://github.com/hYdos/Lint/pull/69/checks?check_run_id=2302394094#step:5:68)

Reply:
Fixed in accessible    method    net/minecraft/world/gen/feature/TreeFeature    isVine    (Lnet/minecraft/world/TestableWorld;Lnet/minecraft/util/math/BlockPos;)Z

Reply:
Fixed in 0abce5bfc576a42e9f67b0dd05e76f8e6f967c86